layouts/partials/blocks/social.html

```diff
@@ -0,0 +1,49 @@
+{{ with .block }}
+  <nav
+    class="mx-5 mt-10 rounded border border-g-4 p-6 shadow md:mx-auto md:mt-16 md:max-w-screen-lg md:px-20 md:py-10"
+    data-ga-category="follow"
+  >
+    <h3
+      class="font-sans text-base font-bold uppercase leading-none tracking-wider text-orange"
```

Comment:
We don't really use text-orange many places anymore. Maybe this should be a different color?